src/node/core/config/types.ts

```diff
@@ -179,7 +179,14 @@ export interface PkgConfigOptions {
   dist?: string
   exports?: PkgConfigProperty<PkgExports>
   extract?: {
-    bundledPackages?: string[]
+    /**
+     * Packages in `devDependencies` that are not in `external` are automatically added to the `bundledPackages` config.
+     * You can exclude a package from being bundled by using a callback:
+     * ```
+     * bundledPackages: (prev) => prev.filter(package => package !== 'sanity')
+     * ```
+     */
+    bundledPackages?: PkgConfigProperty<string[]>
```

Comment:
This PR reads as breaking to me, but from what I can see its already like this but not for types?

Reply:
The change only affects generated typings.
It shouldn't be breaking, but it might be experienced as breaking if:
- a package is relying on the published package exporting types that are implicitly any and that now ship defined types.
- a package is using implicit peer dependencies on typings that can't overlap (for example a `class` with private properties).

In the latter case the fix is to make the implicit peer dependency explicit :)